### PR TITLE
Notify GH upon successful deployment

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

**Comment by amitaibu** [comment]
Can you please show a screenshot of how it looks

**Comment by AronNovak** [comment]
image

**Comment by amitaibu** [comment]
Thanks. Maybe we can make the `qa` from above a link?

File: RoboFile.php [hunk 1]
```diff
@@ -1200,6 +1206,77 @@ protected function githubApiGet(string $path) {
...
+      }
+    }
+
+    if (empty($issue_number)) {
+      $this->say("Giving up, no notification sent to GitHub");
+      return;
+    }
+
+    $this->say("Notifying GitHub of the deployment");
+    $exit_code = $this->taskExec("curl -X POST -H 'Authorization: token $github_token' -d '{\"body\": \"The latest merged PR just got deployed successfully to Pantheon `$pantheon_environment` environment\"}' https://api.github.com/repos/" . self::$githubProject . "/issues/$issue_number/comments")
```
Comment (amitaibu) [inline RoboFile.php:1271]: That would appear on the issue? Maybe link to the PR?

Comment (AronNovak) [reply RoboFile.php:1271]: Yep, we post it to the issue comments. I will try to make it friendlier with links as you suggested.

Comment (AronNovak) [reply RoboFile.php:1271]: (First at the given client project, where I can test too.)

Comment (AronNovak) [reply RoboFile.php]: Pushed the code for this addition.

File: RoboFile.php [hunk 2]
```diff
@@ -1245,6 +1258,90 @@ protected function githubApiGet(string $path) {
...
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+
+    $this->say("Notifying GitHub of the deployment");
+    if (empty($pr_number)) {
+      $issue_comment = "{\"body\": \"The latest merged PR just got deployed successfully to Pantheon [`$pantheon_environment`](https://" . $domain . "/) environment\"}";
+    }
+    else {
+      $issue_comment = "{\"body\": \"The latest merged [PR](https://github.com/" . self::$githubProject . "/pull/" . $pr_number . ") just got deployed successfully to Pantheon [`$pantheon_environment`](https://" . $domain . "/) environment\"}";
```
Comment (mariano-dagostino) [inline RoboFile.php:1334]: @AronNovak What about?

```suggestion
      $issue_comment = "{\"body\": \"The latest merged PR [#$pr_number](https://github.com/" . self::$githubProject . "/pull/" . $pr_number . ") just got deployed successfully to Pantheon [`$pantheon_environment`](https://" . $domain . "/) environment\"}";
```

**Review by amitaibu — APPROVED** [review]

**Pull request merged**